e2e/volumegroupsnapshot.go

```diff
@@ -117,3 +117,65 @@ func (c *cephFSVolumeGroupSnapshot) ValidateResourcesForDelete() error {
 
 	return nil
 }
+
+type rbdVolumeGroupSnapshot struct {
+	*volumeGroupSnapshotterBase
+}
+
+var _ VolumeGroupSnapshotter = &rbdVolumeGroupSnapshot{}
+
+func newRBDVolumeGroupSnapshot(f *framework.Framework, namespace,
```

Comment:
This function creates volumegroupsnapshot but the function name is not self-explanatory, so I think it will be good to have a comment here.

Reply:
maybe, but it follows the same naming as CephFS.